tests/Method/CopyMessageMethodTest.php

```diff
@@ -20,7 +21,6 @@ public function testCreate(): void
             'parseMode' => HasParseModeVariableInterface::PARSE_MODE_MARKDOWN_V2,
             'disableNotification' => true,
             'replyToMessageId' => 0,
-//                    'allowSendingWithoutReply' => true,
```

Comment:
What's going on?

Reply:
whoops, will fix

Reply:
I checked, It's fixel in the last commits https://github.com/tg-bot-api/bot-api-base/pull/44/files#diff-9140e67e920de792745d93cdf7b1d89e21f4122c206263c6479fa45f961942b8